### PR TITLE
Added Extension icon

### PR DESCRIPTION
**Pull request by coliff**
Added icon
Extension icon provided by Christian Oliff (https://github.com/coliff)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,4 +70,4 @@
     "uppercase",
     "vscode"
   ]
-}
+}
```
Comment (ruiquelhas) [inline package.json]: Keep the newline at end of file. Avoid doing it in a new commit, just amend the existing one.

**Review by ruiquelhas — DISMISSED** [review]

**Comment by coliff** [comment]
I used the online editor to make the change and it did it in a separate commit - sorry about that.

**Comment by coliff** [comment]
Sidenote; It'd be good to add an .editorconfig to the project so the end of file line break would be set as a preference, something like...

```
# editorconfig.org

root = true

[*]
charset = utf-8
end_of_line = lf
indent_size = 2
indent_style = space
trim_trailing_whitespace = true

[*.js]
insert_final_newline = true

[*.json]
insert_final_newline = true

[*.md]
trim_trailing_whitespace = false
```

**Comment by ruiquelhas** [comment]
It's just too much boilerplate for something that will probably remain unchanged for some time.

**Pull request merged**